packages/node/test/global-setup.jest.ts

```diff
@@ -0,0 +1,63 @@
+import TicTacToeApp from "@counterfactual/apps/build/TicTacToeApp.json";
+import MinimumViableMultisig from "@counterfactual/contracts/build/MinimumViableMultisig.json";
+import ProxyFactory from "@counterfactual/contracts/build/ProxyFactory.json";
+import dotenvExtended from "dotenv-extended";
+import { ContractFactory, Wallet } from "ethers";
+import { Web3Provider } from "ethers/providers";
+import fs from "fs";
+import ganache from "ganache-core";
+import mkdirp from "mkdirp";
+import os from "os";
+import path from "path";
+
+dotenvExtended.load();
+
+const DIR = path.join(os.tmpdir(), "jest_ganache_global_setup");
+
+module.exports = async () => {
+  const server = ganache.server({
+    accounts: [
+      {
+        balance: "120000000000000000",
+        secretKey: process.env.PRIVATE_KEY_A
+      },
+      {
+        balance: "120000000000000000",
+        secretKey: process.env.PRIVATE_KEY_B
+      },
+      {
+        balance: "120000000000000000",
+        secretKey: process.env.PRIVATE_KEY_C
+      }
+    ]
+  });
+  // @ts-ignore
```

Comment:
Reasoning behind this?

Reply:
TS complains that a property being set on the `Global` object doesn't exist (`Property 'ganacheServer' does not exist on type 'Global'`) even though expected behavior for Jest is to be able to set properties like this.

So ignoring the false TS warning.